magik-language-server/client-vscode/package.json

```diff
@@ -223,15 +223,15 @@
 					"type": "string"
 				},
 				"magik.productDirs": {
-					"description": "Paths to (compiled, containing a `libs/` directory) products",
+					"description": "Paths to (compiled, containing a `libs/` directory) products.",
 					"type": "array"
 				},
 				"magik.aliases": {
-					"description": "Path to gis_aliases file.",
+					"description": "Path to gis_aliases file -- ignored by language server.",
 					"type": "string"
 				},
 				"magik.environment": {
-					"description": "Path to environment file.",
+					"description": "Path to environment file --ignored by language server.",
```

Comment:
Perhaps instead of `ignored by language server`, specify where it is (only) used. I.e., something like `only used by vscode extension`.